src/main/java/org/gridsuite/modification/server/modifications/BranchStatusModification.java

```diff
@@ -41,79 +46,87 @@ public void apply(Network network, Reporter subReporter) {
         switch (modificationInfos.getAction()) {
             case LOCKOUT:
-                applyLockoutBranch(subReporter, branch, branchType);
+                applyLockoutBranch(subReporter, branch, branchTypeName);
                 break;
             case TRIP:
-                applyTripBranch(subReporter, branch, branchType, network);
+                applyTripBranch(subReporter, branch, branchTypeName, network);
                 break;
             case SWITCH_ON:
-                applySwitchOnBranch(subReporter, branch, branchType);
+                applySwitchOnBranch(subReporter, branch, branchTypeName);
                 break;
             case ENERGISE_END_ONE:
-                applyEnergiseBranchEnd(subReporter, branch, branchType, Branch.Side.ONE);
+                applyEnergiseBranchEnd(subReporter, branch, branchTypeName, Branch.Side.ONE);
                 break;
             case ENERGISE_END_TWO:
-                applyEnergiseBranchEnd(subReporter, branch, branchType, Branch.Side.TWO);
+                applyEnergiseBranchEnd(subReporter, branch, branchTypeName, Branch.Side.TWO);
                 break;
             default:
                 throw NetworkModificationException.createBranchActionTypeUnsupported(modificationInfos.getAction());
         }
     }
 
-    private void applyLockoutBranch(Reporter subReporter, Branch<?> branch, String branchType) {
+    private void applyLockoutBranch(Reporter subReporter, Branch<?> branch, String branchTypeName) {
         if (disconnectAllTerminals(branch)) {
             branch.newExtension(BranchStatusAdder.class).withStatus(BranchStatus.Status.PLANNED_OUTAGE).add();
         } else {
             throw new NetworkModificationException(BRANCH_ACTION_ERROR, "Unable to disconnect all branch ends");
         }
         subReporter.report(Report.builder()
-            .withKey("lockout" + branchType + "Applied")
-            .withDefaultMessage(branchType + " ${id} (id) : lockout applied")
+            .withKey("lockout" + branchTypeName + "Applied")
+            .withDefaultMessage(branchTypeName + " ${id} (id) : lockout applied")
             .withValue("id", branch.getId())
             .withSeverity(TypedValue.INFO_SEVERITY)
             .build());
     }
 
-    private void applyTripBranch(Reporter subReporter, Branch<?> branch, String branchType, Network network) {
+    private void applyTripBranch(Reporter subReporter, Branch<?> branch, String branchTypeName, Network network) {
         var trip = new BranchTripping(branch.getId());
         var switchesToDisconnect = new HashSet<Switch>();
         var terminalsToDisconnect = new HashSet<Terminal>();
         var traversedTerminals = new HashSet<Terminal>();
         trip.traverse(network, switchesToDisconnect, terminalsToDisconnect, traversedTerminals);
 
+        LOGGER.info("Apply Trip on {} {}, switchesToDisconnect: {} terminalsToDisconnect: {} traversedTerminals: {}",
```

Comment:
wrong because we display the same MOULAY762 transformer Id ?
In fact a terminal has no Id / Name, so it's hard to print a debug log. I decided to print the name of the connectable, but it is the same (MOULAY762->terminal1 and MOULAY762->terminal2)

how could we do better ? is it important ?

Reply:
No, it's not very important, it's working fine. It's more to understand a bit